cpp/src/strings/search/find.cu

```diff
@@ -548,10 +686,26 @@ std::unique_ptr<column> contains(strings_column_view const& input,
   }
 
   // benchmark measurements showed this to be faster for smaller strings
-  auto pfn = [] __device__(string_view d_string, string_view d_target) {
-    return d_string.find(d_target) != string_view::npos;
-  };
-  return contains_fn(input, target, pfn, stream, mr);
+  return contains_small_strings_impl(input, target, stream, mr);
+}
+
+std::unique_ptr<table> multi_contains(strings_column_view const& input,
+                                      strings_column_view const& targets,
+                                      rmm::cuda_stream_view stream,
+                                      rmm::mr::device_memory_resource* mr)
```

Comment:
Please combine this with the one above. All this does is wrap a column-vector with a table. There is no reason the other `multi_contains` cannot return a table and handle the threshold logic.

Reply:
updated.